src/frontend/components/VideoCard/VideoCard.js

```diff
@@ -0,0 +1,25 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisVertical } from '@fortawesome/free-solid-svg-icons';
+import { Link } from 'react-router-dom';
+import { getImageUrl } from '../../utils/video-helpers';
+import './VideoCard.css';
+
+export function VideoCard({ videoData }) {
+	const { _id, title, category } = videoData;
+	return (
+		<article className="video-card">
+			<Link to={`/watch/${category}/${_id}`}>
+				<div className="card-media">
+					<img src={getImageUrl(_id)} alt={title} />
```

Comment:
can use lazy loading for images

Reply:
Currently there is no much data in the db, so the images will most likely be on the view port on Page load. Will add a lazy loading component in the future